helm-chart/mezod/values.yaml

```diff
@@ -22,7 +22,10 @@ env:
   MEZOD_PORT_P2P: 26656 # Default. It should be changed in case of a custom setup
 
 # -- Set Secret object containing the keyring information: KEYRING_NAME, KEYRING_PASSWORD, KEYRING_MNEMONIC and ETHEREUM_ENDPOINT
-secret: "NAME_OF_THE_SECRET"
+# -- Set Secret object with node_key.json
+secrets:
+  general_config: "YOUR_SECRET_NAME"
+  node_key: "YOUR_SECRET_NAME"
```

Comment:
Imho it's better to use `credentials` instead of `general_config`, and `nodeKey` instead of `node_key` as we use mostly camelCase for other variables.